aviary/utils/test_utils/default_subsystems.py

```diff
@@ -7,6 +7,17 @@
 
 
 def get_default_premission_subsystems(legacy_code, engines=None):
+    """
+    Get default premission subsystems propulsion, geometry, aerodynamics, and mass
+    in this order.
+
+    Arguments:
+    ----------
+    legacy_code : str
+        either 'FLOPS' or 'GASP'
```

Comment:
```suggestion
    legacy_code : str, LegacyCode
        either FLOPS or GASP LegacyCode Enums, or their strings equivalents ('FLOPS', 'GASP')
```
The LegacyCode Enum can also be directly passed to this function and it will work!